blocks/OSC/src/cinder/osc/Osc.cpp

```diff
@@ -863,7 +867,7 @@ std::ostream& operator<<( std::ostream &os, const Message &rhs )
 {
 	os << "Address: " << rhs.getAddress() << std::endl;
 	if( ! rhs.getSenderIpAddress().is_unspecified() )
```

Comment:
Note that `address::is_unspecified()` is not properly implemented in the [ASIO version Cinder uses](https://github.com/cinder/Cinder/blob/5586c04e9e07e09a269e30fdd357ed5329911616/include/asio/ip/impl/address.ipp#L201-L212) -- it will return `false` for an empty address (or any address with type `none`, as opposed to `ip4` or `ip6`).

I see later versions of ASIO [have fixed this](https://github.com/chriskohlhoff/asio/blob/b0926b61b057ce563241d609cae5768ed3a4e1b1/asio/include/asio/ip/impl/address.ipp#L195-L200).  Anyhow, not a big deal here -- we'll continue to assume the method returns the right thing, as the previous code did. The normal thing for users is to output a `Message` you just received, which **will** have the sender data filled in properly (it will be "specified").